lib/pom.rb

```diff
@@ -212,8 +203,8 @@ def installer.ensure_required_ruby_version_met; end
           end
         end
 
-        if g.default_spec
-          specfile_wildcard = "#{g.name}-#{version}*.gemspec"
+        if true # default_spec was always true for these?
```

Comment:
Why leave this `if` here?

Reply:
yeah I did this last night and did not notice this until after I submitted but the intent looked like mkristian planned on this being run by more than default at some point.  I will remove though.